src/core/raster/qgshillshaderenderer.h

```diff
@@ -102,11 +108,18 @@ class CORE_EXPORT QgsHillshadeRenderer : public QgsRasterRenderer
      */
     void setZFactor( double zfactor ) { mZFactor = zfactor; }
 
+    /**
+     * @brief Set IsMultiDirection
+     * @param isMultiDirection Is multi direction
+     */
+    void SetMultiDirection( bool isMultiDirection ){ mMultiDirection = isMultiDirection; }
```

Comment:
Or we could use a `setMethod( methodEnum )` to prepare for future methods? This would of course require changes to more parts of the code.